editor-std/ui_data_uk.txt

```diff
@@ -81,17 +81,17 @@ Zoom Out = Зменьшити
 Fit in Window = На все вікно
 Refresh = Поновити
 Snap = Прив\'язка
-Cross Cursor = Плюсовий курсор
+Cross Cursor = Курсор у вигляді хреста
 Toolbars = Панелі інструментів
 Drawing Tools = Інструменти малювання
 Wiring Tools = Інструменти розведення
-Power Objects = Потужні обьекты
+Power Objects = Об\'єкти потужності
```

Comment:
Hi, does it need the \ charater?

Reply:
I think it's not needed.

Reply:
I'm not sure. Because some translations by other users had it. Anyway, i always will correct the translation.